src/general/OptionsMenu.tscn

```diff
@@ -295,6 +297,41 @@ text = "None"
 items = [ "None", null, false, 0, null, "Protanope (Red-Green)", null, false, 1, null, "Tritanope (Blue-Yellow)", null, false, 2, null ]
 selected = 0
 
+[node name="HBoxContainer5" type="HBoxContainer" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Graphics/VBoxContainer"]
+margin_top = 237.0
+margin_right = 517.0
+margin_bottom = 262.0
+size_flags_horizontal = 3
+
+[node name="ChromaticToggle" type="CheckBox" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Graphics/VBoxContainer/HBoxContainer5"]
+margin_right = 16.0
+margin_bottom = 25.0
+pressed = true
+
+[node name="Label" type="Label" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Graphics/VBoxContainer/HBoxContainer5"]
+margin_left = 20.0
+margin_top = 1.0
+margin_right = 235.0
+margin_bottom = 24.0
+text = "Chromatic Aberration:"
+
+[node name="HSeparator" type="HSeparator" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Graphics/VBoxContainer/HBoxContainer5"]
+margin_left = 239.0
+margin_right = 313.0
+margin_bottom = 25.0
+size_flags_horizontal = 3
+custom_styles/separator = SubResource( 4 )
+
+[node name="ChromaticAmount" type="HSlider" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Graphics/VBoxContainer/HBoxContainer5"]
```

Comment:
It seems like you didn't hook up the chromatic aberration sliders to immediately send values to Settings like the other setting widgets do